audio.py

```diff
@@ -88,3 +135,35 @@ def compute_duration(audio_path, backend = 'ffmpeg'):
 	cmd = ['ffprobe', '-v', 'error', '-show_entries', 'format=duration', '-of', 'default=noprint_wrappers=1:nokey=1'
 			] if backend == 'ffmpeg' else ['soxi', '-D'] if backend == 'sox' else None
 	return float(subprocess.check_output(cmd + [audio_path]))
+
+
+def timeit(audio_path, numbers, sample_rate, mono, audio_backend, delimiter):
+	start_process_time = time.process_time_ns()
+	start_perf_counter = time.perf_counter_ns()
+	for i in range(numbers):
+		signal, sample_rate = read_audio(audio_path, sample_rate = sample_rate, mono = mono, backend = audio_backend)
+	end_process_time = time.process_time_ns()
+	end_perf_counter = time.perf_counter_ns()
+	process_time = (end_process_time - start_process_time) / delimiter
+	perf_counter = (end_perf_counter - start_perf_counter) / delimiter
+	print(audio_path, numbers, audio_backend, process_time, perf_counter)
+
+
+if __name__ == '__main__':
+	import argparse
+
+	parser = argparse.ArgumentParser()
+	subparsers = parser.add_subparsers()
+	cmd = subparsers.add_parser('timeit')
+
+	cmd.add_argument('--audio-path', type = str, required = True)
+	cmd.add_argument('--sample-rate', type = int, default = 8000)
+	cmd.add_argument('--mono', action = 'store_true')
+	cmd.add_argument('--audio-backend', type = str, required = True)
+	cmd.add_argument('--numbers', type = int, default = 100)
+	cmd.add_argument('--delimiter', type = int, default = 1000000)
```

Comment:
Shouldn't this be 1000? 1 microsecond is 1000 nanoseconds

Reply:
Final result in table in ms, so I choose delimeter 1000000.

Reply:
chaned to 1000